public/content/developers/docs/mev/index.md

```diff
@@ -20,6 +20,8 @@ In theory MEV accrues entirely to validators because they are the only party tha
 
 Validators do get a portion of the full MEV amount anyway because searchers are willing to pay high gas fees (which go to the validator) in exchange for higher likelihood of inclusion of their profitable transactions in a block. Assuming searchers are economically rational, the gas fee that a searcher is willing to pay will be an amount up to 100% of the searcher's MEV (because if the gas fee was higher, the searcher would lose money).
 
+![Screenshot from 2024-04-24 00-30-49](https://github.com/ethereum/ethereum-org-website/assets/99739569/a6a92c68-32db-4730-979b-0b2dae79b47a)
```

Comment:
I see there is discussion in-progress over on the issue (#12691) but when we do settle on a diagram, please note we should 

- Save and rename the image file inside `public/content/developers/docs/mev/`
- Reference the file locally, without the "Screenshot" alt, ie: `![MEV Diagram](./mev-diagram.png)`

Reply:
Ok no problems Paul when a conclusion is reach to settle for a diagram i will make a pull request to resolve this.
Thanks